spec/factories/user_submitted_source_records.rb

```diff
@@ -15,7 +15,7 @@
         {
           "id": "10.1016/j.petrol.2016.02.026",
           "type": "doi",
-          "url": "http://dx.doi.org/10.1016/j.petrol.2016.02.026",
+          "url": "https://dx.doi.org/10.1016/j.petrol.2016.02.026",
```

Comment:
Is a fixture with an `http` prefix also required?  It could test what happens when a user enters an `http` instead of `https` protocol, to check whether or not the application will use the `http` as is or coerce it into an `https`.